seed/swift-sdk/simple-fhir/Sources/Schemas/Account.swift

```diff
@@ -55,6 +55,10 @@ public struct Account: Codable, Hashable, Sendable {
         try container.encodeIfPresent(self.practitioner, forKey: .practitioner)
     }
 
+    public enum Account: String, Codable, Hashable, CaseIterable, Sendable {
+        case account = "Account"
```

Comment:
@kafkas does it make more sense to do `case Value = "Account"` ?

Reply:
Good question. I went with derived case names for readability and intent.

For example:

```swift
let request = CreateAccountRequest(type: .account)
```

In Swift, when the type is already known from context, it’s idiomatic to use the shorthand member syntax `.account` instead of writing the full `Account.account` and presumably our customers will be doing this in the vast majority of cases. If we used `.value`, you’d see:

```swift
let request = CreateAccountRequest(type: .value)
```